dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/StoredCompanyRepository.kt

```diff
@@ -56,7 +56,7 @@ interface StoredCompanyRepository : JpaRepository<StoredCompanyEntity, String> {
         "SELECT DISTINCT company.countryCode FROM StoredCompanyEntity company " +
             "LEFT JOIN company.dataRegisteredByDataland data " +
             "WHERE " +
-            "data.dataType NOT IN ('lksg')"
+            "data.dataType NOT IN ('lksg', 'sfdr')"
```

Comment:
Currently writing test to detect if this is done.
In this test you can set for which frameworks you expect filter options to be generated, and assure that it works.